fdbserver/ClusterController.actor.cpp

```diff
@@ -1719,6 +1736,11 @@ class ClusterControllerData {
 				} else if (regions[0].dcId == clusterControllerDcId.get()) {
 					return reply.get();
 				}
+				TraceEvent(SevWarn, "CCRecruitmentFailed", id)
+				    .detail("Reason", "Recruited Txn system and CC are in different DCs")
+				    .detail("CCDcID", clusterControllerDcId.get())
```

Comment:
To be consistent with the other events here:

```suggestion
				    .detail("CCDcId", clusterControllerDcId.get())
```